numba/cuda/tests/cudadrv/test_cuda_devicerecord.py

```diff
@@ -85,6 +86,7 @@ class TestCudaDeviceRecordWithRecord(TestCudaDeviceRecord):
     Tests the DeviceRecord class with np.record host types
     """
     def setUp(self):
+        super().setUp()
```

Comment:
This is now calling `TestCudaDeviceRecord.setUp()`, which wasn't the case previously - the idea here is that `TestCudaDeviceRecordWithRecord` has a distinct `setUp` method. So I think this needs to be:

```suggestion
        CUDATestCase.setUp()
```

in order to bypass `TestCudaDeviceRecord.setUp()`.